tool/cli.js

```diff
@@ -891,6 +893,61 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .argument("<output>", "Output to file or stdout", {
+    default: "stdout",
+  })
+  .action(
+    tryOrExit(async ({ args, options }) => {
+      const { verbose } = options;
+      const { output } = args;
+
+      if (verbose) {
+        console.log(chalk.green("Checking for precense of CONTENT_ROOT"));
+      }
+
+      if (!CONTENT_ROOT) {
+        throw new Error(
+          "CONTENT_ROOT not set. Please see the docs at tool/README.md"
```

Comment:
I realize now that this won't make sense running the content repo. There is no `tool/README.md` file in that repo. Either this ought to be a URL, or we ought to use [Caporal's `help`](https://caporal.io/guide/commands.html#help) to provide the help text via the `--help` option.